src/sonic-config-engine/minigraph.py

```diff
@@ -124,6 +124,7 @@ def parse_dpg(dpg, hname):
 
         ipintfs = child.find(str(QName(ns, "IPInterfaces")))
         intfs = []
+        intfnames = {}
```

Comment:
I think it is to align with the other designs, e.g. portchannels, vlans, and aclintfs. I am okay with this.